Liberation.fontdef

```diff
@@ -0,0 +1,15 @@
+font Liberation/Sans
```

Comment:
please move the new files to a fonts/ directory

Reply:
Perhaps they could go in a zip file

Reply:
actually, I would prefer a folder for easier editing & git history